src/components/Button/Button.jsx

```diff
@@ -30,11 +30,9 @@ const Button = ({ size, color, outline, disabled, onClick, type, className,
 
 Button.propTypes = {
   /** The size of the button */
-  size: propTypes.string,
+  size: propTypes.oneOf(['default', 'large']),
   /** The color of the button. */
-  color: propTypes.string,
-  /** Whether the button is an outline only, or solid. */
-  outline: propTypes.bool,
```

Comment:
@dankolbman I would investigate whether there are outline buttons still in use in the wild. If there are then they need to documented by @kcullion in the ui guidelines before this gets deprecated.

Reply:
Updated the description with the changes. Only coordinator is affected here.

Reply:
when deprecating from the outline shouldn't the fill color be the same as the outline color was? In your example in the description it seems the button color changes which can be confusing to users as functionality should be encoded by color

Reply:
The previous implementation used a different color as a default as the color palette naming wasn't correct. This was also corrected in this pr.

User wise, this only impacts internal users atm.